.github/workflows/build.yml

```
@@ -16,11 +16,17 @@ jobs:
               ARCHIVE_NAME: 'gcc-2.7.2-linux.tar.gz'
             }
           - {
-              OS: 'macos-latest',
-              CFLAGS: '-DDARWIN -std=gnu89 -Wno-return-type -Wno-error -Wno-implicit-function-declaration',
+              OS: 'macos-13',
+              CFLAGS: '-DDARWIN -std=gnu89 -Wno-return-type -Wno-error -Wno-implicit-function-declaration -Wno-int-conversion',
               HOST: 'i386-apple-darwin',
               ARCHIVE_NAME: 'gcc-2.7.2-mac.tar.gz'
             }
+          - {
+              OS: 'macos-latest',
```

Looking at the github actions runner repo (https://github.com/actions/runner-images?tab=readme-ov-file#available-images) seems like the meaning of macos-latest has shifted to an arm machine.
It would be better to pin the images we use to avoid confusion, so what about using `macos-14` here?
```suggestion
              OS: 'macos-14',
```